great-mystery-great-whore/original/067-chapter-66.adoc

```diff
@@ -81,6 +81,10 @@ In such the man of sin reigns, and in such the devil may transform;
 but the elect he cannot deceive, the elect have the victory and dominion,
 who were elected in him before the foundation of the world.
 Glory forever!
+
+[.small-break]
```

Comment:
This was with the paragraph above and I just happened to see it. I moved it down on it's own with a small break, don't know if that was the right thing to do!

Reply:
seems good to me 👍